<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicFeatureSettingsDialog</class>
 <widget class="QDialog" name="BasicFeatureSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>158</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Basic feature settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="title">
       <property name="toolTip">
        <string>Channel marker title</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="colorLayout">
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="colorBtn">
       <property name="toolTip">
        <string>Channel marker color</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="colorText">
       <property name="text">
        <string>#ff0000</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="reverseAPILayout">
     <item>
      <widget class="QCheckBox" name="reverseAPI">
       <property name="toolTip">
        <string>Sychronize with reverse API </string>
       </property>
       <property name="text">
        <string>reverse API</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reverseAPIAddress">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reverse API address</string>
       </property>
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reverseAPISeparator">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reverseAPIPort">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reverse API port</string>
       </property>
       <property name="inputMask">
        <string>00000</string>
       </property>
       <property name="text">
        <string>8888</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reverseAPIFeatureSetLabel">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reverseAPIFeatureSetIndex">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reverse API destination feature set index</string>
       </property>
       <property name="inputMask">
        <string>00</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reverseAPIFeatureLabel">
       <property name="text">
        <string>Feat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reverseAPIFeatureIndex">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reverse API destination feature index</string>
       </property>
       <property name="inputMask">
        <string>00</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BasicFeatureSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BasicFeatureSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
